vtex-apps/search
# feat/increment-events: Incrementing autocomplete events

## What is the purpose of the request?
Increment native autocomplete component events.

## How to test?
Go to the `feat/increment-events` branch.
Run `vtex link`.
Run a search on the `search autocomplete v2` component.
Check the return of the `pixelManagerEvents` variable.

## Which events were incremented?
`autocomplete`, `store.search // history_click` and others.

## Attachments
![image](https://user-images.githubusercontent.com/32168339/226035904-b5ff8a39-ee78-46a9-9df1-a197dbb3b103.png)
